### DISCUSSION TITLE
Slash Commands

### DISCUSSION THREAD

**Discussion by Deadtrix21** [Pycord-Development/pycord#524] [Q&A] [answered]
How do they function
Because i just cant find the slash command

is there something that i need to do to make the work or what

**Comment by Lulalaby** [comment, marked as answer]
The best would be that you look into our examples and join our official support server:
 https://discord.gg/pycord

**Reply by Deadtrix21** [reply]
thats the thing i copied it 
and pasted 

and i can't get the slash commands to how up in discord 

```py
 @slash_command()
    async def hi(self, ctx):
        await ctx.respond(f"Hi, this is a global slash command from a cog!")
```

and i am porting from discord.py

**Reply by RGBCube** [reply]
use 
```py
@slash_command(guild_ids=[paste your server id here])
    async def hi(self, ctx):
        await ctx.respond(f"Hi, this is a global slash command from a cog!")```

**Reply by Deadtrix21** [reply]
tried it and nothing

**Reply by Deadtrix21** [reply]
let say this is my guild id 751905819611758662
how would it go into that

**Comment by ImNimboss** [comment]
well its not global if it has the `guild_ids` parameter lol

**Reply by ImNimboss** [reply]
sorry, I accidentally answered instead of replied

**Reply by Deadtrix21** [reply]
see i am trying to use it where ever but no where where i check it shows up
like no where at all

**Comment by Lulalaby** [comment]
As I already said, please join the support guild. It's easier to help there
